edgecast/waf/access_rule_models.go

```diff
@@ -79,6 +79,10 @@ type AccessControls struct {
 	Whitelist []interface{} `json:"whitelist"`
 }
 
+func NewAddAccessRuleParams() *AddAccessRuleParams {
```

Comment:
Do the parameters returned here need to be a pointer? I notice many functions in the sdk take pointers for parameters, even though the parameters themselves are never expected to be nil, or even handle a possible nil case.

Reply:
@defstream good catch here. I updated the methods in the PR to not return a pointer as it is not needed. I will add a new ticket to clean this up in the rest of the SDK.